webapp/general/general.py

```diff
@@ -0,0 +1,25 @@
+import psycopg as dbm
+
+from flask import Blueprint, render_template, current_app
+
+from utils import groupby_query_rows
+
+general_bp = Blueprint('general_bp', __name__,
+                       template_folder='templates')
+
+@general_bp.route('/')
+def main_page():
+    with dbm.connect(current_app.config["DB_URI"]) as con:
+        cur = con.cursor()
+        # extracting data for tiles on mane page
+        cur.execute("""SELECT 
```

Comment:
Does your execute command return a correct result? I think it should be checked here.

Reply:
Resolved